### ISSUE TITLE
Some of the user's avatars don't exist anymore.

### ISSUE THREAD

**Issue by lakesare** [lakesare/memcode#49]
Users sign up via github/google, and, when they remove their avatars, we get this look:

![image](https://user-images.githubusercontent.com/7578559/77825168-662c5280-7129-11ea-938d-281c712e7a2d.png)

As soon as user signs up - we should download their image to our amazon, and store that link instead.

**Comment by anandvenkat4** [comment]
Instead, why don't we update the avatar each time they login?

**Comment by lakesare** [comment]
Some users won't log in again (☠️), - and this will be a bad look in any place where we'd like to show their avatar (in the user select e.g., soon we'll add the user list dropdown under the '10 students learn this course' stat)

**Comment by anandvenkat4** [comment]
Got it. In that case, are we presenting an option(in future) for the users to change their profile picture fro our platform?

**Comment by lakesare** [comment]
Absolutely, here is the issue for that functionality: https://github.com/lakesare/memcode/issues/22.
